stable/mysql/5.7.20/values.yaml

```diff
@@ -0,0 +1,65 @@
+_config:
+  _metadata:
+    name: mysql
+    version: 5.7.20
+    description: |
+      Fast, reliable, scalable, and easy to use open-source relational database.
+  controllers:
+  - containers:
+    - env:
+      - name: MYSQL_ROOT_PASSWORD
+        value: root
+      - name: MYSQL_DATABASE
+        value: db
+      - name: MYSQL_USER
+        value: user
+      - name: MYSQL_PASSWORD
+        value: password
+      mounts:
+      - path: "/var/lib/mysql"
+        name: data
+      command: []
+      args: []
+      imagePullPolicy: Always
+      image: cargo.caicloudprivatetest.com/library/mysql:5.7.20
```

Comment:
How do we manage different image repo?

Reply:
https://github.com/caicloud/charts/blob/master/build/charts/build.sh#L36

Build script would replace it.

Reply:
images in library are synced from upstream like docker hub.

Reply:
OK